operations/payments.md

```diff
@@ -401,6 +403,59 @@ Returns all payments in the system, filtered by various parameters. At least one
 | `Payments` | array of [Payment](#payment) | required | The list of filtered payments. |
 | `Cursor` | string | required | Unique identifier of the last and hence oldest payment returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of payments. |
 
+## Refund payment
+
+Refunds a specified payment, only alternative or credit card payments can be refunded. For more information about refund use [Get all payments](payments.md#get-all-payments) using `RefundId`. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
```

Comment:
```suggestion
Refunds a specified payment. Note only credit card or alternative payments can be refunded. For more information about the refunded payment, use [Get all payments](#get-all-payments) with `PaymentId`. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
```
Note @VBuben I assume this should be `PaymentId` and not `RefundId` - can you confirm that?

Reply:
Yes you are right. What I was meaning by my terrible writting was use `RefundId` paremter from this endpoint and use the value in [Get all payments](#get-all-payments) into `PaymentIds` field